packages/e2e-tests/specs/editor/various/multi-block-selection.test.js

```diff
@@ -747,4 +748,87 @@ describe( 'Multi-block selection', () => {
 		} );
 		expect( selectedText ).toEqual( 'Post title' );
 	} );
+
+	it( 'should multi-select in the ListView component with shift + click', async () => {
```

Comment:
There is also a list-view.test.js, not sure if the tests should be there or here. I think this is fine 😄